firmware/watchguard_sysa_dl.ksy

```diff
@@ -0,0 +1,137 @@
+meta:
+  id: watchguard_sysa_dl
+  endian: be
+  title: Watchguard Fireware web update (Sysa-dl) file format
+  license: MIT
```

Comment:
`ksylint` is actually right on both points:

```console
$ ksylint watchguard_sysa_dl.ksy
watchguard_sysa_dl.ksy
137:23: no new line character at the end of file (new-line-at-end-of-file)
Keys are not ordered: ['id', 'endian', 'title', 'license']
```

The "Keys are not ordered" is unfortunately not vert user-friendly, because it lists the _existing_ order of keys and assumes that you immediately know the correct one. That may not be the case - in such case, see https://doc.kaitai.io/ksy_style_guide.html#meta:

For top-level meta section, use the following order of keys:

1. <ins>**`id`**</ins> — main identifier, MUST match the name of the `.ksy` file
2. Human-readable meta-information (in order of importance, at least one MUST be present):
   a. <ins>**`title`**</ins>
   (...)
3. Legal information — <ins>**`license`**</ins> — MUST be a valid [SPDX license expression](https://spdx.org/licenses/)
4. (...)
5. Defaults (in alphabetic order):
   a. `encoding`
   b. <ins>**`endian`**</ins>